src/pages/404.jsx

```diff
@@ -1,20 +1,29 @@
 import * as React from 'react';
 import { Link } from 'gatsby';
+import { StaticImage } from 'gatsby-plugin-image';
 
 // styles
 const pageStyles = {
-  color: '#232129',
+  backgroundColor:"#121212",
```

Comment:
This is not how we normally handle switching between light and dark modes, please change this